misc-scripts/ontology/sql/tables.sql

```diff
@@ -38,100 +37,98 @@ INSERT INTO meta (meta_key, meta_value)
+  `ontology_id` int(10) unsigned NOT NULL,
+  `subsets` text,
+  `accession` varchar(64) NOT NULL,
+  `name` varchar(255) NOT NULL,
+  `definition` text,
+  `is_root` int(11) NOT NULL DEFAULT '0',
+  `is_obsolete` int(11) NOT NULL DEFAULT '0',
+  `iri` text,
+  PRIMARY KEY (`term_id`),
+  UNIQUE KEY `accession` (`accession`),
+  UNIQUE KEY `term_ontology_acc_idx` (`ontology_id`,`accession`),
+  KEY `term_name_idx` (`name`(100))
+) ENGINE=MyISAM DEFAULT CHARSET=utf8;
+
+CREATE TABLE `synonym` (
+  `synonym_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `term_id` int(10) unsigned NOT NULL,
+  `name` text CHARACTER SET utf8 NOT NULL,
+  `type` enum('EXACT','BROAD','NARROW','RELATED') COLLATE utf8_unicode_ci DEFAULT NULL,
+  `dbxref` varchar(500) COLLATE utf8_unicode_ci DEFAULT NULL,
+  PRIMARY KEY (`synonym_id`),
+  UNIQUE KEY `synonym_term_idx` (`term_id`,`synonym_id`),
+  KEY `synonym_name_idx` (`name`(100))
+) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
+
+CREATE TABLE `alt_id` (
+  `alt_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `term_id` int(10) unsigned NOT NULL,
+  `accession` varchar(64) NOT NULL,
+  PRIMARY KEY (`alt_id`),
+  UNIQUE KEY `term_alt_idx` (`term_id`,`alt_id`),
+  KEY `ix_alt_id_accession` (`accession`)
+) ENGINE=MyISAM DEFAULT CHARSET=utf8;
+
+CREATE TABLE `relation_type` (
+  `relation_type_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(64) NOT NULL,
+  PRIMARY KEY (`relation_type_id`),
+  UNIQUE KEY `name` (`name`)
+) ENGINE=MyISAM DEFAULT CHARSET=utf8;
+
+
+CREATE TABLE `relation` (
+  `relation_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `child_term_id` int(10) unsigned NOT NULL,
+  `parent_term_id` int(10) unsigned NOT NULL,
+  `relation_type_id` int(10) unsigned NOT NULL,
+  `intersection_of` tinyint(1) NOT NULL,
+  `ontology_id` int(10) unsigned NOT NULL,
+  PRIMARY KEY (`relation_id`),
+  UNIQUE KEY `child_parent_idx` (`child_term_id`,`parent_term_id`,`relation_type_id`,`intersection_of`,`ontology_id`),
+  KEY `ix_relation_parent_term_id` (`parent_term_id`),
+  KEY `ix_relation_relation_type_id` (`relation_type_id`),
+  KEY `ix_relation_ontology_id` (`ontology_id`)
+) ENGINE=MyISAM DEFAULT CHARSET=utf8;
+
+
+CREATE TABLE `closure` (
+  `closure_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `child_term_id` int(10) unsigned NOT NULL,
+  `parent_term_id` int(10) unsigned NOT NULL,
+  `subparent_term_id` int(10) unsigned DEFAULT NULL,
+  `distance` tinyint(3) unsigned NOT NULL,
+  `ontology_id` int(10) unsigned NOT NULL,
+  `confident_relationship` tinyint(1) NOT NULL,
+  PRIMARY KEY (`closure_id`),
+  UNIQUE KEY `closure_child_parent_idx` (`child_term_id`,`parent_term_id`,`subparent_term_id`,`ontology_id`),
+  KEY `ix_closure_subparent_term_id` (`subparent_term_id`),
+  KEY `ix_closure_ontology_id` (`ontology_id`),
+  KEY `parent_subparent_idx` (`parent_term_id`,`subparent_term_id`)
+) ENGINE=MyISAM DEFAULT CHARSET=utf8;
```

Comment:
New indexes for table closure should also be in the patch

Reply:
Done.